lib/Parse/DNS/Zone.pm

```diff
@@ -493,7 +544,11 @@ sub _parse_zone {
 		}
 
 		$origin = $1, next if(/^\$ORIGIN ([\w\-\.]+)\s*$/i);
-		$def_ttl = $1, next if(/^\$TTL (\d+)\s*$/i);
```

Comment:
I would restore this line, and just changing it to do `_parse_time_unit($1)` instead. Any reason not to?

Reply:
I'm not used to this syntax and PBP doesn't recommend it: https://metacpan.org/pod/Perl::Critic::Policy::ValuesAndExpressions::ProhibitCommaSeparatedStatements